src/freertos_drivers/esp32/Esp32HardwareCanAdapter.hxx

```diff
@@ -129,7 +132,9 @@ private:
 
     /// Enables/Disables the periodic reporting of CAN bus statistics to the
     /// default serial stream.
-    bool reportStats_;
+    bool reportStats_ : 1;
+    /// Set to true of the frame sropped warning is printed.
+    bool warningPrinted_ : 1;
```

Comment:
Perhaps naming as overrunWarningPrinted to clarify it's usage further?

Reply:
good idea. done.